coprocess/python/tyk/middleware.py

```diff
@@ -16,7 +17,8 @@ def __init__(self, filepath):
         self.handlers = {}
 
         try:
-            self.module = import_module(filepath)
+            source = SourceFileLoader('middleware', self.filepath)
```

Comment:
Is it always going to be "middleware"? (Just nitpicking)

Reply:
This is only used internally, it could be set to "x" and would still work.